py-polars/polars/expr/string.py

```diff
@@ -1681,14 +1681,19 @@ def slice(self, offset: int, length: int | None = None) -> Expr:
         offset
             Start index. Negative indexing is supported.
         length
-            Length of the slice. If set to ``None`` (default), the slice is taken to the
-            end of the string.
+            Length in characters of the slice. If set to ``None`` (default), the slice
+            is taken to the end of the string.
 
         Returns
         -------
         Expr
             Expression of data type :class:`Utf8`.
 
+        Notes
```

Comment:
I also added this note to both the str and expr docstrings for `slice`.

Reply:
I wanted to be explicit in the definition as per @orlp 's comment [here](https://github.com/pola-rs/polars/issues/10349#issuecomment-1668073918), but I wonder if we should remove the "non-surrogate" part, since not many people will recognize what that means.

Reply:
Without it it's technically not correct. You can also call it a Unicode Scalar Value like the Rust docs if you prefer.